src/terminal.c

```diff
@@ -320,6 +373,11 @@ libedit_private void
 terminal_end(EditLine *el)
 {
 
+        if (dl_handle != NULL) {
+                dlclose(dl_handle);
+                dl_handle = NULL;
+	}					        }
```

Comment:
There is one more brace "}".

Reply:
Thank you for your comment!
I missed it. Probably mistake from ^C/^V action.